static/sass/styles.scss

```diff
@@ -166,6 +166,7 @@ $color-shadow: rgba(0, 0, 0, 0.5) !default;
 @include ubuntu-u-vertically-center;
 @include ubuntu-p-dell;
 @include ubuntu-p-tube-line;
+@include ubuntu-p-detailed-list;
```

Comment:
if we are included 'ubuntu-p-lists' above, do we need this?

Reply:
We do not, thanks for catching!